dockstore-client/pom.xml

```diff
@@ -245,6 +245,19 @@
             <groupId>com.google.code.findbugs</groupId>
             <artifactId>findbugs</artifactId>
         </dependency>
+
+        <dependency>
```

Comment:
Is there an explanation or something for this? 
Link as code comment for maintaining pom.xml

Reply:
Funny story actually, it's from I think an issue created in another repo.  I scroll down and I see references to your commit.